cmake/RINGMesh.cmake

```diff
@@ -143,6 +153,7 @@ install(
 
 # Exports RINGMesh target
 include(CMakePackageConfigHelpers)
+set(CMAKE_INSTALL_SYSTEM_RUNTIME_DESTINATION bin/${CMAKE_BUILD_TYPE})

```

Comment:
Yes, it is only for windows. However, the syntax is not correct. https://cmake.org/cmake/help/v3.1/variable/CMAKE_CFG_INTDIR.html